app/services/curation_concerns/lock_manager.rb

```diff
@@ -28,7 +28,7 @@ def lock(key)
       def uri
         @uri ||= begin
           opts = options
```

Comment:
I don't believe this `opts = options` is getting us anything (except an additional local variable assignment). Do you have time to tweak this?